steno/src/model/stream/media_thread.cpp

```diff
@@ -209,12 +209,15 @@ void MediaThread::run()
             AudioChunk audioChunk;
             while (audioSource_->readAudioChunk(audioChunk) && readCount > 0)
             {
-                std::vector<int> sourcesToSuppress =
-                    Classifier::classify(sourcePositions, imagePositions, classifierRangeThreshold);
+                uint8_t audioChunkFiltered[audioChunk.size];
+                std::copy(audioChunk.audioData.get(), audioChunk.audioData.get() + audioChunk.size, audioChunkFiltered);
```

Comment:
Pass the audio chunk to the function and I don't think the copy is necessary (careful with stack arrays)